util/cron/test-perf.chapcs.cc.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Run -perflabel cc- performance tests on a chapcs machine with gasnet
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='chapcs'
```

Comment:
I don't really know the impact of these variables but I can't tell if this should be chapcs-comm-counts, say